app/views/events/_material_area.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if can? :view_material, Event %>
-  <div class="col-xs-12 col-sm-12 col-md-10 col-lg-10 col-xs-offset-0 col-sm-offset-0 col-md-offset-1 col-lg-offset-1" id="event_material_area">
+  <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12" id="event_material_area">
```

Comment:
Yip i use ZEn-Coding, so the classes were added automatically, but I see your point